usr/share/rear/conf/default.conf

```diff
@@ -818,39 +818,59 @@ CLONE_ALL_USERS_GROUPS="no"
+# (e.g. if the secure shell software loads special libraries via dlopen).
+#
 # When SSH_FILES is set to a 'false' value no SSH file is copied into the recovery system
 # and nothing is done to set up SSH (in particular no sshd) in the recovery system.
 # In this case one could still use generic ReaR functionality like COPY_AS_IS and
 # REQUIRED_PROGS to manually copy some plain SSH files into the recovery system
-# but then there is still none of the special SSH setup in the recovery system.
-# When SSH_FILES is set to a 'true' value all the "usual SSH files" in
-# /etc/ssh* /etc/openssh* /etc/centrifydc/ssh* /root/.ssh /root/.shosts
-# (including private SSH keys) get copied into the recovery system
-# to make things "just work" in the recovery system which usually makes
-# the recovery system (in particular its ISO image and any recovery medium)
+# but then there is still none of the special SSH setup for the recovery system.
+#
+# When SSH_FILES is set to a 'true' value the "usual OpenSSH >= 3.1 files" in the
+# /etc/ssh /root/.ssh /root/.shosts directories are copied into the recovery system
+# including private keys provided SSH_UNPROTECTED_PRIVATE_KEYS is 'true' (see below).
+# This makes remote access "just work" in the recovery system but (usually) this also
+# makes the recovery system (in particular its ISO image and any recovery medium)
 # a confidential piece of data that needs to be protected against unwated access
 # cf. https://github.com/rear/rear/issues/1511 and https://github.com/rear/rear/issues/1512
+#
 # When SSH_FILES is an array of files or directories like SSH_FILES=( /etc/ssh /root/.ssh )
 # those files and directories are copied into the recovery system via COPY_AS_IS
 # so that also COPY_AS_IS_EXCLUDE works to exclude some unwanted files from that list
 # like COPY_AS_IS_EXCLUDE=( "${COPY_AS_IS_EXCLUDE[@]}" '/etc/ssh/sh_host_*' '/root/.ssh/id_*' )
-# where in particular excluding '/etc/ssh/*key*' results that SSH host key files
-# will be generated anew in the recovery system as described below.
+# where in particular excluding '/etc/ssh/sh_host_*' results that SSH host key files
+# will be generated anew for the recovery system (see below).
+#
 # When SSH_FILES is not set to a 'false' value additionally some SSH programs
 # are copied into the recovery system, in particular ssh, sshd, scp, sftp, ssh-agent,
 # and ssh-keygen plus a sftp-server program (e.g. /usr/lib/ssh/sftp-server).
+#
 # The default setting SSH_FILES='avoid_sensitive_files' should be reasonably secure
 # because it avoids copying sensitive SSH files as follows:
 # From /etc/ssh only moduli, ssh_config, sshd_config, and ssh_known_hosts are copied and
 # from /root/.ssh only authorized_keys and known_hosts are copied into the recovery system.
 # SSH host key files can be generated anew only for the recovery system
 # when SSH_UNPROTECTED_PRIVATE_KEYS is not set to a 'false' value (which is not the default).
```

Comment:
The triple negation ("not", "false", "not") makes this sentence somewhat hard to read.

Suggestion:
> SSH host key files will be generated anew for the recovery system if SSH_UNPROTECTED_PRIVATE_KEYS is either to a 'true' value or a pattern list ('false' being the default).

The same applies to subsequent occurrences.